x-pack/plugins/aiops/public/embeddable/embeddable_change_point_chart.tsx

```diff
@@ -58,12 +58,30 @@ export class EmbeddableChangePointChart extends AbstractEmbeddable<
 
   private node?: HTMLElement;
 
+  deferEmbeddableLoad = true;
```

Comment:
nit: Suggestion to make a note why we need to deferEmbeddableLoad, similar like how did below for AnomalyDetectionEmbeddable

Reply:
added in https://github.com/elastic/kibana/pull/164485/commits/6218319d92247ce337f6fd80167cf52b7c5f8741